Adyen/Validators/Validator.swift

```diff
@@ -22,3 +22,56 @@ public protocol Validator {
     func maximumLength(for value: String) -> Int
     
 }
+
+/// :nodoc:
+public func || (lhs: Validator, rhs: Validator) -> Validator {
+    ORValidator(lhs: lhs, rhs: rhs)
+}
+
+/// :nodoc:
+public func && (lhs: Validator, rhs: Validator) -> Validator {
+    ANDValidator(lhs: lhs, rhs: rhs)
+}
+
+/// Interface to allow for two validators.
+internal protocol CombinedValidator: Validator {
```

Comment:
why do we need this protocol? it doesn't add any value IMO

Reply:
It was to avoid duplicating some parts but it only ended up having `func maximumLength(for value: String) -> Int`.
May remove it

Reply:
I see